src/scss/_mixins.scss

```diff
@@ -0,0 +1,41 @@
+@mixin font-weight($weight) {
+  font-weight: $weight;
+}
+
+@mixin font-size($size) {
+  font-size: $size;
+}
+
+@mixin margin($top, $right, $bottom, $left) {
+  margin-top: $top;
+  margin-right: $right;
+  margin-bottom: $bottom;
+  margin-left: $left;
+}
```

Comment:
I love the switch from margin variables to a margin mixin.  Much cleaner.  Great job!